src/um/propkey.rs

```diff
@@ -0,0 +1,2309 @@
+// Copyright © 2016-2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+
+// NOTE: The pointless imports are required to make the tests not complain
+//       about guiddef and wtypes not being used byt this module.
+
+#[allow(unused_imports)]
+use shared::guiddef::LPGUID;
+use shared::minwindef::DWORD;
+use shared::ntdef::{DOUBLE, LONG};
+#[allow(unused_imports)]
+use shared::wtypes::BSTR;
+
+DEFINE_PROPERTYKEY!(PKEY_Audio_ChannelCount,
+    0x64440490, 0x4C8B, 0x11D1, 0x8B, 0x70, 0x08, 0x00, 0x36, 0xB1, 0x1A, 0x03, 7);
+pub const AUDIO_CHANNELCOUNT_MONO: DWORD = 1;
+pub const AUDIO_CHANNELCOUNT_STEREO: DWORD = 2;
+DEFINE_PROPERTYKEY!(PKEY_Audio_Compression,
+    0x64440490, 0x4C8B, 0x11D1, 0x8B, 0x70, 0x08, 0x00, 0x36, 0xB1, 0x1A, 0x03, 10);
+DEFINE_PROPERTYKEY!(PKEY_Audio_EncodingBitrate,
+    0x64440490, 0x4C8B, 0x11D1, 0x8B, 0x70, 0x08, 0x00, 0x36, 0xB1, 0x1A, 0x03, 4);
+DEFINE_PROPERTYKEY!(PKEY_Audio_Format,
+    0x64440490, 0x4C8B, 0x11D1, 0x8B, 0x70, 0x08, 0x00, 0x36, 0xB1, 0x1A, 0x03, 2);
+DEFINE_PROPERTYKEY!(PKEY_Audio_IsVariableBitRate,
+    0xE6822FEE, 0x8C17, 0x4D62, 0x82, 0x3C, 0x8E, 0x9C, 0xFC, 0xBD, 0x1D, 0x5C, 100);
+DEFINE_PROPERTYKEY!(PKEY_Audio_PeakValue,
+    0x2579E5D0, 0x1116, 0x4084, 0xBD, 0x9A, 0x9B, 0x4F, 0x7C, 0xB4, 0xDF, 0x5E, 100);
+DEFINE_PROPERTYKEY!(PKEY_Audio_SampleRate,
```

Comment:
`DEFINE_PROPERTYKEY` is an item level macro and should use `{}` instead of `()`.